cumulus/pallets/parachain-system/src/validate_block/tests.rs

```diff
@@ -316,15 +322,15 @@ fn validate_block_works_with_child_tries() {
 
 	let parent_head = block.header().clone();
 
-	let TestBlockData { block, validation_data } = build_block_with_witness(
+	let TestBlockData { block, validation_data, slot } = build_block_with_witness(
 		&client,
 		vec![generate_extrinsic(&client, Alice, TestPalletCall::read_and_write_child_tries {})],
 		parent_head.clone(),
 		Default::default(),
 	);
 
-	let header = block.header().clone();
-
+	let block = seal_block(block, slot, &client);
+	let header: cumulus_test_runtime::Header = block.header().clone();
```

Comment:
Can type be removed? Was rustc complaining?

Reply:
Yep, removed!